docs/resources/supported_operations.md

```diff
@@ -0,0 +1,1072 @@
+# Supported Operations {#openvino_resources_supported_operations}
+
+@sphinxdirective
+
+
+This page lists operations supported by OpenVINO. The two tables you can find below present basic information.
+For more detailed and most recent listing of operations that are implemented and tested:
+
+.. button-link:: _static/download/operation_conformance_table_files/opset_report_omz_static.html
+   :color: primary
+   :outline:
+
+   See the full conformance report table
+
+
+.. tab-set::
+
+   .. tab-item:: Operations Alphabetically
+
+      =================================  ==============  ===============  ================  ==================
+       Operations                         GPU             CPU              GNA               Arm® CPU    
+      =================================  ==============  ===============  ================  ==================  
```

Comment:
May be the order should be CPU, GPU, GNA? @dmitry-gorokhov, @vladimir-paramuzov, please advice. Also, shoul we have ARM here?

Reply:
I think the devices order is not that important. What really matters is that operation names and supported flags are outdated. To my mind we need to update this table based on the latest conformance report.

Reply:
I think it is a good idea. The order will reflect the ops coverage - most covered first, least covered last... I don't know about Arm, but since it differs from the standard CPU I would leave it. Just change the name to "CPU (Arm®)"